eng/pipelines/templates/steps/sdk-testgen-set-env.yml

```diff
@@ -9,7 +9,6 @@ steps:
       version: '1.22.1'
     displayName: "Install Go"
```

Comment:
Unless this template is reused then we should probably eliminate this entire step template and move the pieces inline to the ci.yml.

Reply:
Removed.